src/engraving/dom/note.cpp

```diff
@@ -549,7 +549,7 @@ NoteHeadGroup NoteHead::headGroup() const
 //---------------------------------------------------------
 
 Note::Note(Chord* ch)
-    : EngravingItem(ElementType::NOTE, ch, ElementFlag::MOVABLE)
+    : EngravingItem(ElementType::NOTE, ch, ElementFlag::MOVABLE), m_jumpPoints(this)
```

Comment:
Unless I'm mistaken, I don't think it's necessary to do this in the constructor? You could do this directly in the declaration:
`TieJumpPointList m_jumpPoints = TieJumpPointList(this);`